tmt/convert.py

```diff
@@ -709,6 +718,93 @@ def read_nitrate(
 
 
+def read_polarion(
+        data: NitrateDataType,
+        polarion_case_id: Optional[str],
+        link_polarion: bool) -> None:
+    """ Read data from Polarion """
+    # Find Polarion case
+    echo(style('Polarion ', fg='blue'), nl=False)
+    polarion_case = tmt.export.get_polarion_case(data, polarion_case_id=polarion_case_id)
+    if not polarion_case:
+        raise ConvertError('Failed to find test case in Polarion')
+
+    # Update summary
+    if not data.get('summary') or data['summary'] != polarion_case.title:
+        data['summary'] = str(polarion_case.title)
+        echo(style('summary: ', fg='green') + data['summary'])
+
+    # Update description
+    if polarion_case.description:
+        data['description'] = str(polarion_case.description)
+        echo(style('description: ', fg='green') + data['description'])
+
+    # Update status
+    status = True if polarion_case.status == 'approved' else False
+    if not data.get('enabled') or data['enabled'] != status:
+        data['enabled'] = status
+        echo(style('enabled: ', fg='green') + str(data['enabled']))
+
+    # Update assignee
+    if polarion_case.assignee:
+        data['contact'] = str(polarion_case.assignee[0].name).replace(
+            '(', '<').replace(')', '@redhat.com>')
+        echo(style('contact: ', fg='green') + data['contact'])
+
+    # Set tmt id if available in Polarion, otherwise generate
+    try:
+        UUID(polarion_case.test_case_id)
+        uuid = str(polarion_case.test_case_id)
+    except (ValueError, TypeError):
+        uuid = str(uuid4())
+        polarion_case.test_case_id = uuid
+        polarion_case.update()
+    data[tmt.identifier.ID_KEY] = uuid
+    echo(style('ID: ', fg='green') + uuid)
+
+    # Update component
+    if polarion_case.casecomponent:
+        data['component'] = []
+        for component in polarion_case.casecomponent:
+            data['component'].append(str(component))
+        echo(style('component: ', fg='green') + ' '.join(data['component']))
+
+    # Update tags
+    if polarion_case.tags:
+        if not data.get('tag'):
+            data['tag'] = []
+        for tag in polarion_case.tags.split():
+            data['tag'].append(tag)
+            tier_match = re.match(r'^Tier ?(?P<num>\d+)$', tag, re.I)
+            if tier_match:
+                num = tier_match.group('num')
+                if 'tier' in data:
+                    if int(num) < int(data['tier']):
+                        log.warning('Multiple Tier tags found, using the one with a lower index')
+                        data['tier'] = num
+                else:
+                    data['tier'] = num
+                echo(style('tier: ', fg='green') + str(data['tier']))
+        data['tag'] = sorted(set(data['tag']))
+        echo(style('tag: ', fg='green') + ' '.join(data['tag']))
+
+    # Add Polarion links for Requirements and the case
+    if link_polarion:
+        server_url = str(polarion_case._session._server.url)
```

Comment:
What about handling the missing/present slash at the end here instead of doing it twice below?